lib/modules/hosts/imgur.js

```diff
@@ -219,7 +219,6 @@ const imgur = {
 			type: 'VIDEO',
 			controls: false,
 			href: ShowImages.module.options.preferredImgurLink.value === 'share' ? `${imgur.baseURL}${info.id}` : `${info.link}`,
-			downloadurl: info.download,
```

Comment:
This would be a good place to fill in the downloadurl if it's  not sent anymore, e.g. `info.download || info.sources[0].source`